src/main/java/org/aksw/simba/squirrel/deduplication/hashing/impl/ArrayHashValue.java

```diff
@@ -0,0 +1,79 @@
+package org.aksw.simba.squirrel.deduplication.hashing.impl;
+
+import org.aksw.simba.squirrel.deduplication.hashing.HashValue;
+
+/**
+ * A hash value as a Array of integers.
+ */
+public class ArrayHashValue implements HashValue {
```

Comment:
I am not sure whether we have a real advantage when we store the hash value as an array of integer values...
Comparing integers is much faster than a string which would represent these integers. However, since it might be easier to directly "search" for a particular hash value in Redis, a consistent representation of the hashes (as hex values) might be better.